jme3-examples/src/main/java/jme3test/awt/TestCanvas.java

```diff
@@ -172,6 +172,10 @@ public void actionPerformed(ActionEvent e) {
                 currentPanel.remove(canvas);
                 app.stop(true);
 
+                try {
+                    Thread.sleep(1000);

```

Comment:
This is a test yes but looks like a hack of sorts. Better at least to comment why this is needed. Should consumer add this to their production code?

Reply:
> Should consumer add this to their production code?

It is an important part for lwjgl3-awt to work correctly (the consumer will have to implement it), in the pending commits I plan to document this part.

Reply:
> It's better to have "LwjglCanvas" handle this internally (I'll put it on my list of things to fix).

Definitely. Optimally users should not care whether they are running LWJGL 2 or 3. Identical code should work. Any shenanigans should be hidden from the user (exist in the library itself).